src/main/java/com/marek/onlinebookstore/controller/ShoppingCartController.java

```diff
@@ -50,6 +50,7 @@ public ShoppingCartDto addItemToUserCart(Authentication authentication,
         return shoppingCartService.addItem(principal, requestDto);
     }
 
+    @ResponseStatus(HttpStatus.OK)
     @PutMapping("/cart-items/{id}")
```

Comment:
The @ResponseStatus(HttpStatus.OK) annotation is not necessary for the PUT request method since HTTP 200 OK is the default response status for PUT. If there is a specific reason for explicitly declaring the status, consider adding a comment to explain.